src/tab.js

```diff
@@ -130,7 +130,11 @@ SideTab.prototype = {
     this._setIcon("img/defaultFavicon.svg");
   },
   setSpinner() {
-    this._setIcon("img/loading-spinner.svg");
+    // Workaround until context-stroke is enabled in sidebar by default.
+    const isDarkTheme = document.body.classList.contains("dark-theme");
+    const isShrinked = document.getElementById("tablist").classList.contains("shrinked");
+    const frag = isDarkTheme && isShrinked && "#loading-invert" || "";
```

Comment:
It is needed.
Until `context-stroke` is enabled by default, the fallback color in SVG is used.
```
#loading-invert {
  stroke: context-stroke rgb(245, 247, 250);
}
#loading {
  stroke: context-stroke HighLight;
}
```